library/src/main/res/values-v9/strings.xml

```diff
@@ -2,5 +2,5 @@
 <!-- Copyright 2012 Square, Inc. -->
 
 <resources>
-  <string name="month_name_format">LLLL yyyy</string>
+  <string name="month_name_format" translatable="false">LLLL yyyy</string>
```

Comment:
I think we want to keep this translatable, actually (so if other languages want to do "year month" or something, they can).  You can just leave it un-translated in your new strings files, though.